testpmd/hooks/install.yml

```diff
@@ -1,12 +1,48 @@
 ---
+- name: Get all RuntimeClass
+  community.kubernetes.k8s_info:
+    api_version: node.k8s.io/v1
+    kind: RuntimeClass
+  register: rclass
+
+- name: Select High performance RuntimeClass
+  set_fact:
+    runtime_class: "{{ rclass | json_query(query) | first }}"
+  vars:
+    query: "resources[?handler=='high-performance'].metadata.name"
+  when: rclass|length > 0
 
 - name: Deploy the Example CNF applications
   include_role:
     name: "{{ deploy_dir }}/roles/example-cnf-app"
+  vars:
+    high_perf_runtime: "{{ runtime_class }}"
```

Comment:
runtime_class may not be defined (there's a when clause in the previous task). This runtime_class, I believe, should have a default value (empty string, for example)

Reply:
This is not fixed yet, I think the line you've added in the PodDisruptionBudget task should be here in fact.